k8s/demo/crunchy-postgres/run.sh

```diff
@@ -18,22 +18,22 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-$DIR/cleanup.sh
+#$DIR/cleanup.sh
 
 # create the service account used in the containers
 kubectl create -f $DIR/set-sa.json
 
 # as of Kube 1.6, we need to allow the service account to perform
```

Comment:
For version Kube 1.6, we must allow the service account to perform a label command. For this example, OpenEBS opens up wide permissions for all service accounts. This is NOT for production!